weaver/processes/wps_package.py

```diff
@@ -2069,7 +2069,8 @@ def make_location_input(self, input_type, input_definition):
                         f"Directory reference resolution method for input [{input_id}] "
                         f"from location [{input_location}] did not produce any staged file."
                     )
-                input_location = out_dir
+                input_directory_name = os.path.basename(os.path.normpath(input_definition.url))
```

Comment:
Should reuse `get_secure_directory_name`.

Reply:
Good spot!